### PR TITLE
Add Integration Resource

### PR DESCRIPTION
**Pull request by erikzaadi**
# Description

What - Add a new Integration resource

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: port/integration/schema.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		"version": schema.StringAttribute{
+			Optional: true,
+		},
+		"title": schema.StringAttribute{
+			Optional: true,
+		},
+		"installation_app_type": schema.StringAttribute{
+			Required:   true,
+			Validators: []validator.String{stringvalidator.OneOfCaseInsensitive("WEBHOOK", "KAFKA")},
+		},
```
Comment (Tankilevitch) [inline port/integration/schema.go:30]: by default users don't know of it, and actually I am less familiar with that as well, therefore I wonder if we want to have it optional with default.

File: port/integration/resource_test.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+						}]
+					}
+				}
+			}]
+		})
+		webhook_changelog_destination = {
+			type = "WEBHOOK"
+			url = "https://google.com"
+			agent = true
+		}
```
Comment (Tankilevitch) [inline port/integration/resource_test.go:50]: lets have a test without `webhook_changelog_destination` as well

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
Nicely done, great initiative with the `CONTRIBUTING.md`

File: Makefile [hunk 3]
```diff
@@ -30,9 +30,6 @@ release:
 clean:
 	rm -rf examples/.terraform examples/.terraform.lock.hcl examples/terraform*
 
-run-example:
-	cd examples && terraform init && terraform apply
-
```
[the remark this answers is not in the document]
Comment (erikzaadi) [reply Makefile:35]: In the previous example refactor I changed it so that each example is ran per folder

File: port/integration/schema.go (hunk 1 above)
Comment (erikzaadi) [reply port/integration/schema.go:30]: This is required in the schema, what would be the default then?

Comment (erikzaadi) [reply port/integration/schema.go:30]: It's actually not required by the API, making optional

**Review by Tankilevitch — APPROVED** [review]
LGTM

**Pull request merged**